nixos/modules/services/networking/mosh.nix

```diff
@@ -0,0 +1,19 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg  = config.services.mosh;
+
+in
+{
+  options.services.mosh = {
+      enable = mkEnableOption "mosh";
+  };
+
+  config = mkIf cfg.enable {
+    environment.systemPackages = with pkgs; [ mosh ];
+    networking.firewall.allowedUDPPortRanges = [ { from = 60000; to = 61000; } ];
```

Comment:
Can this port range be configured in mosh? Can it be configured globally? Maybe this should be configurable from this module.